modules/ROOT/pages/relnotes-java-sdk.adoc

```diff
@@ -28,7 +28,7 @@ The other dependency is RxJava which is both used internally and exposed as a fi
 
 == Version 2.7.2 (5 December 2018)
 
-Version 2.7.2 is the third release of the 2.7 series, bringing new features, enhancements, and bugfixes over the last stable release.
+Version 2.7.2 is the third release of the 2.7 series, bringing new features, enhancements, and bugfixes over the last stable release. As of this release, OpenJDK 11 is fully supported. See the xref:compatibility-versions-features.adoc[Compatibility] section for details.
```

Comment:
Both OpenJDK 11 and Oracle JDK 11.

Also, you should mention with the HotSpot JVM.

I'd remove the word "fully", as it implies there is some other partial support otherwise.

Reply:
Sure, fixed.